src/actions/WalletActions.tsx

```diff
@@ -270,10 +270,9 @@ const activateWalletTokens = async (
     const feeString = `${displayFee} ${feeDenom.name} (${fiatFee} ${fiatCurrencyCode.replace('iso:', '')})`
     let bodyText = s.strings.activate_wallet_token_scene_body
 
-    // HACK: XRP causes the reserve requirement to increase per token activated. There's
-    // No good way to parametrize this and abstract away the asset
-    if (wallet.currencyInfo.pluginId === 'ripple') {
-      bodyText += '\n\n' + s.strings.activate_wallet_token_scene_body_xrp_extra
+    const info = getSpecialCurrencyInfo(wallet.currencyInfo.pluginId)
```

Comment:
Optional: I have been using this structure in new code:

```ts
const { tokenActivationAdditionalReserveText } = SPECIAL_CURRENCY_INFO[pluginId] ?? {}
```

This gives us a chance to assign a default value if we need it. We don't need the `getSpecialCurrencyInfo` helper in this case, since TypeScript understands the `?? {}` bit.